python-sdk/src/transfers/constants.py

```diff
@@ -0,0 +1,27 @@
+from enum import Enum
+
+
+class FileLocation(Enum):
+    # [START filelocation]
+    LOCAL = "local"
+    HTTP = "http"
+    HTTPS = "https"
+    GS = "google_cloud_platform"  # Google Cloud Storage
+    google_cloud_platform = "google_cloud_platform"  # Google Cloud Storage
+    S3 = "s3"  # Amazon S3
+    AWS = "aws"
```

Comment:
Is the plan to keep this static to what we "officially" support? Will there be a way for users to load custom locations/ingestors on top of this library?

Reply:
It would be nice if users could easily create custom ingestors and treat them as plugins